CONTRIBUTING.md

```diff
@@ -229,6 +229,20 @@ Run script to download dashboards to `./dashboards` folder
 node scripts/download_dashboards.mjs
 ```
 
+## Contributing to Documentation
+
+When submitting PRs for documentation updates, build and run the documentation locally to ensure functionality before submission. For first time documentation contributors, install the python dependencies with `yarn docs:install`. Build the documentation locally with `yarn docs:build` and serve with `yarn docs:serve`
```

Comment:
to clarify `yarn docs:build` just builds part of the docs, namely the CLI references, if there are no updates to those I don't think this step is required.

`yarn docs:serve` will watch for changes and hot reload the UI

Reply:
> Might be fine how we documented it here though.

Yea I agree. Just high level description here in `CONTRIBUTING.md`. For the specific command usage, contributors will need to look deeper.

Reply:
I think its a nice to have for those that do not know there are generated sections.  Perhaps a comment that it only needs to be run once but having it noted seems important to me.

Reply:
I think it's always better to include too much than too little, even if one step is not _always_ required, executing it won't hurt, it'll just be redundant, but it covers all cases if they just do it every time.